keps/sig-node/2727-grpc-probe/kep.yaml

```diff
@@ -0,0 +1,18 @@
+title: Add gRPC probe to Pod.Spec.Container.{Liveness,Readiness,Startup}Probe
+kep-number: 2727
+authors:
+  - "@bowei"
+  - "@PxyUp"
+owning-sig: sig-node
+participating-sigs:
+  - sig-node
+  - sig-network
+status: provisional
```

Comment:
This needs to be set to implementable if you want to begin work in 1.22.

Reply:
done